x-pack/plugins/actions/server/lib/gen_ai_token_tracking.test.ts

```diff
@@ -256,6 +256,7 @@ describe('getGenAiTokenTracking', () => {
       })
     );
   });
+
```

Comment:
Wondering if we should create a `gen_ai` subdirectory here, and set CODE_OWNERS so we don't need to review.  Or move to a package.

Reply:
I will do in a different PR! Will link soon...

Reply:
>  I assume none of these sub-actions will be queued, and so don't have to suffer through intermediate releases - still all used "live"?

That's correct!